### DISCUSSION TITLE
meshFromPoint(clientX, clientY)

### DISCUSSION THREAD

**Discussion by thibaud-be** [google/model-viewer#3973] [General]
I'm trying to retrieve the mesh name on click.
pretty much what the materialFromPoint(clientX, clientY) does but for the mesh.
What are my options ?

**Comment by elalish** [comment]
Our API intentionally doesn't expose mesh-level info (we consider that to be at the level of three.js). We recommend authoring the glTF with separate materials at the level of granularity you want for selection & editing.
